utils/config/daos_server.yml

```diff
@@ -601,3 +601,13 @@
 #
 #  bdev_auto_faulty:
 #    enable: false
+#
+## Override the default file transfer mechanism for dmg support collect-log by supplying
+## the path to an alternative script or binary that will be used to copy log off of the servers.
+## Note that the --transfer-args flag to the collect-log command may be used to supply
+## extra runtime arguments used by the copy tool (e.g. a cloud bucket name, etc.)
+##support_config
+##  file_transfer_exec: /usr/bin/copy_tool.sh
+#
+#support_config:
+#  file_transfer_exec:
```

Comment:
Maybe suggest rsync as that's the default:

```suggestion
#support_config:
#  file_transfer_exec: /usr/bin/rsync
```
It would be good to call that out in the documentation comment, too.

Reply:
the problem with this file is that it is somehow used in the tests. I can change the example /usr/bin/copy_tool.sh to /usr/bin/rsync however.